open-api/rest-catalog-open-api.yaml

```diff
@@ -3647,6 +3826,151 @@ components:
             type: integer
           description: "List of equality field IDs"
 
+    DeleteFile:
```

Comment:
Do we need a `discriminator` here?

Reply:
I think so. I think this should be similar to `ContentFile` except that it only has the delete file types.

Also, I think this should be located with ContentFile instead of after the delete file types.

Reply:
Will fix this thanks